applications/ParticleMechanicsApplication/custom_constitutive/hencky_plastic_3d_law.cpp

```diff
@@ -201,8 +201,8 @@ void HenckyElasticPlastic3DLaw::CalculateMaterialResponseKirchhoff (Parameters&
 
     const ProcessInfo& CurrentProcessInfo = rValues.GetProcessInfo();
 
-    const Matrix&   DeformationGradientF   = rValues.GetDeformationGradientF();
-    double DeterminantF                    = rValues.GetDeterminantF();
+    Matrix DeformationGradientF     = rValues.GetDeformationGradientF();
```

Comment:
Are you intending to make a copy?
This might be expensive

Reply:
Hi @philbucher, thanks for the check! I forgot to remove this copy. I didnt want to change the original matrix in the first hand, since I need to convert from 2D-3D-2D for 2d case. But it is fixed now. Cheers!